### ISSUE TITLE
test: TEST-25-IMPORT started suddenly timeouting under qemu

### ISSUE THREAD

**Issue by mrc0mmand** [systemd/systemd#12039]
In several recent PRs the CentOS CI Vagrant job failed due to timeout in TEST-25-IMPORT:
```
         Starting Network Service...
[  OK  ] Started D-Bus System Message Bus.
+ set -o pipefail
+ export SYSTEMD_PAGER=cat
+ SYSTEMD_PAGER=cat
+ dd if=/dev/urandom of=/var/tmp/testimage.raw bs=1048583 count=5
5+0 records in
5+0 records out
5242915 bytes (5.2 MB, 5.0 MiB) copied, 2.31841 s, 2.3 MB/s
+ machinectl import-raw /var/tmp/testimage.raw
         Starting Virtual Machine a… Container Download Service...
[  OK  ] Found device /dev/ttyS0.
Failed to transfer image: Failed to activate service 'org.freedesktop.import1': timed out (service_start_timeout=25000ms)
[FAILED] Failed to start Testsuite service.
See 'systemctl status testsuite.service' for details.
[  OK  ] Started Login Service.
         Starting End the test...

```

Full log: https://ci.centos.org/job/systemd-pr-build-vagrant/777/artifact/systemd-centos-ci/artifacts_66jQcA/vagrant-logs.rTA/vagrant-arch-testsuite.eSW/TEST-25-IMPORT_FAIL.log
Artifacts: https://ci.centos.org/job/systemd-pr-build-vagrant/777/artifact/systemd-centos-ci/index.html

It doesn't happen every time, but for the past few days it failed for majority of the PRs (see https://ci.centos.org/job/systemd-pr-build-vagrant/). I'm not sure if this is just yet another *qemu-peformance-related* issue or a real bug.

**Comment by mrc0mmand** [comment]
I thought it was a timing issue, but apparently it's not. Did some bisecting, but I'm still clueless about the root cause.

Notes, so faR:

Bisect log:
```
git bisect start
# good: [e62a7fea757f259eb330da5b6d3ab4ede46400a2] Merge pull request #11714 from poettering/final-news-241
git bisect good e62a7fea757f259eb330da5b6d3ab4ede46400a2
# bad: [fa28e4e37778a825fca19a912ae478994c217e5c] Merge pull request #12059 from poettering/nspawn-typos
git bisect bad fa28e4e37778a825fca19a912ae478994c217e5c
# bad: [672d1f57f7fda9f9e6518668e90d35df7ee14c2d] Merge pull request #11863 from poettering/subvol-no-submount
git bisect bad 672d1f57f7fda9f9e6518668e90d35df7ee14c2d
# good: [4da33154524541c2e6477fc837b8e38465c9c9e9] test-network: add testcases for handling disable_ipv6 sysctl property
git bisect good 4da33154524541c2e6477fc837b8e38465c9c9e9
# good: [200fb167a2e5353d0ac6e1f8a5d89bbad3842692] Merge pull request #11844 from keszybz/networkd-fuzzer-fixes
git bisect good 200fb167a2e5353d0ac6e1f8a5d89bbad3842692
# bad: [8a1ef3bf8d0eaea5a34911b3a34d741ef3fb4b5c] systemctl: use SYNTHETIC_ERRNO() where appropriate
git bisect bad 8a1ef3bf8d0eaea5a34911b3a34d741ef3fb4b5c
# bad: [2fe4c28d3068ea92da3b384e5a660757fbb4964b] Merge pull request #11834 from martinpitt/network-test-fixes
git bisect bad 2fe4c28d3068ea92da3b384e5a660757fbb4964b
# bad: [4aa4c4b0f42cfbaa39b6559090cebcca27ddbc8d] network: avoid address section freed
git bisect bad 4aa4c4b0f42cfbaa39b6559090cebcca27ddbc8d
# bad: [9cb8c5593443d24c19e40bfd4fc06d672f8c554c] network: rename GatewayOnlink= to GatewayOnLink=
git bisect bad 9cb8c5593443d24c19e40bfd4fc06d672f8c554c
# good: [703bc7a2a67af315d2515b55d06eba63cf6ba9f1] test-network: drop relevant ip routing policy rules before testing
git bisect good 703bc7a2a67af315d2515b55d06eba63cf6ba9f1
# bad: [b677774d6958435bead38202e61509139b06b497] test-network: add testcase for issue #11280
git bisect bad b677774d6958435bead38202e61509139b06b497
# first bad commit: [b677774d6958435bead38202e61509139b06b497] test-network: add testcase for issue #11280
```

git log
```
* | 358fb6862b - (3 weeks ago) fuzz: do not assume the existence of /sys/class/net/lo - Yu Watanabe
|/
*   dc16327c48 - (3 weeks ago) Merge pull request #11795 from yuwata/fix-network-routing-policy-11280 - Lennart Poettering
|\
| * b677774d69 - (4 weeks ago) test-network: add testcase for issue #11280 - Yu Watanabe (HEAD, refs/bisect/bad)
| * 703bc7a2a6 - (3 weeks ago) test-network: drop relevant ip routing policy rules before testing - Yu Watanabe (refs/bisect/good-703bc7a2a67af315d2515b55d06eba63cf6ba9f1)
| * 031fb59a98 - (4 weeks ago) network: do not remove rule when it is requested by existing links - Yu Watanabe
| * 92cd00b974 - (4 weeks ago) network: remove routing policy rule from foreign rule database when it is removed - Yu Watanabe
* | d9e15cbd18 - (5 weeks ago) journalctl: New option --cursor-file - Jörg Sommer
* |   200fb167a2 - (3 weeks ago) Merge pull request #11844 from keszybz/networkd-fuzzer-fixes - Lennart Poettering (refs/bisect/good-200fb167a2e5353d0ac6e1f8a5d89bbad3842692)
|\ \
| * | c448459d56 - (3 weeks ago) networkd: refuse more than 128 NTP servers - Zbigniew Jędrzejewski-Szmek
| * | 83ec459276 - (3 weeks ago) networkd: fix memleak when the same NetDev is specified twice - Zbigniew Jędrzejewski-Szmek
| * | 3772cfde03 - (3 weeks ago) network: wrap some long lines - Zbigniew Jędrzejewski-Szmek
* | |   4b151b7132 - (3 weeks ago) Merge pull request #11807 from yuwata/test-vlan-mtu - Zbigniew Jędrzejewski-Szmek
|\ \ \

```

**Comment by mrc0mmand** [comment]
See https://github.com/systemd/systemd/pull/12026#issuecomment-478364372
